tests/storage/largeblock/test_largeblock_sr.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from lib.common import wait_for, vm_image
+from tests.storage import try_to_create_sr_with_missing_device, vdi_is_open
+
+# Requirements:
+# - one XCP-ng host with an additional unused 4KiB disk for the SR
+
+class TestLARGEBLOCKSRCreateDestroy:
+    """
+    Tests that do not use fixtures that setup the SR or import VMs,
+    because they precisely need to test SR creation and destruction,
+    and VM import.
+    """
+
+    def test_create_sr_with_missing_device(self, host):
+        try_to_create_sr_with_missing_device('largeblock', 'LARGEBLOCK-local-SR-test', host)
+
+    def test_create_and_destroy_sr(self, host, sr_disk_4k):
+        # Create and destroy tested in the same test to leave the host as unchanged as possible
+        sr = host.sr_create('largeblock', "LARGEBLOCK-local-SR-test", {'device': '/dev/' + sr_disk_4k}, verify=True)
+        # import a VM in order to detect vm import issues here rather than in the vm_on_xfs_fixture used in
+        # the next tests, because errors in fixtures break teardown
+        vm = host.import_vm(vm_image('mini-linux-x86_64-bios'), sr_uuid=sr.uuid)
```

Comment:
If VM import fails, then the SR is not destroyed. Someday we should fix this, but it's not specific to this test. Just mentioning it because I see it.